test/cli/commands_test.py

```diff
@@ -798,6 +798,17 @@ def test__cli__command__fix(rule, fname):
             ],
             1,
         ),
+        (
+            """
+            CREATE TABLE IF NOT EXISTS vuln.software_name_dictionary(
+            id SERIAL PRIMARY KEY,
+            rule VARCHAR(30),
+            );
+            """,
+            ["--force", "--dialect", "postgres", "--disable_progress_bar", "--nocolor"],
+            None,
+            1
+        )
     ],
```

Comment:
What's the relevance of this test case to the change you're making? I can't see that this test checks anything with stderr or stdout.

Reply:
I just wanted to ensure that the formatting of this text still used the exit code of 1 when trying to format this, which would symbolize an error and that this would make its way to stderr. I can add another test that checks the result of stderr vs stdout if you would like?

Reply:
@alanmcruickshank  I have removed this and added a format test to ensure that the file does not get populated with stderr/out messages. Thank you for the feedback and the great software.